spec/controllers/v1/responses_controller_spec.rb

```diff
@@ -97,7 +97,96 @@
       expect(questions.first["key_question"]).not_to be_nil
       expect(questions.first["key_question"]["points"].count).to eq(3)
     end
+  end
+
+
+  context '#show_slimmed' do
+    before do
+      Response.create(responder_id:   @participant.id,
+                      responder_type: 'Participant',
+                      id: 42)
+      create_struct
+    end
+
+    it 'assigns rubric' do
+      get :show_slimmed, assessment_id: assessment.id, response_id: 42
+      expect(assigns(:rubric)[:id]).to eq(@rubric.id)
+    end
+
+    it 'shows a users response' do
+      get :show_slimmed, assessment_id: assessment.id, response_id: 42
+      expect(json["id"]).to eq(42)
+    end
 
+    it "doesn't allow non-owners to view response" do
+      sign_in @user3
+      get :show_slimmed, assessment_id: assessment.id, response_id: 42
+      assert_response :forbidden
+    end
+
+    it 'returns the category => question => score struct' do
```

Comment:
Probably a domain misunderstanding, but how are you testing the ` => question => score struct` of this?